crates/stackable-telemetry/src/instrumentation/axum/mod.rs

```diff
@@ -242,6 +286,28 @@ impl RequestExt for Request {
     }
 }
 
+fn server_host_to_tuple(
+    host: &str,
+    scheme: Option<&str>,
+) -> Result<(String, u16), ServerHostError> {
+    if let Some((host, port)) = host.split_once(':') {
+        // First, see if the host header value contains a colon indicating that
+        // it includes a non-default port.
+        let port: u16 = port.parse().context(ParsePortSnafu)?;
```

Comment:
...following on from a previous suggestion:

```suggestion
        let port: u16 = port.parse().context(ParsePortSnafu { port })?;
```

_untested, maybe there is a borrow issue here_

Reply:
Added in 47ffb40